spec/active_record/oracle_enhanced/type/text_spec.rb

```diff
@@ -241,4 +245,25 @@ class ::TestSerializeEmployee < ActiveRecord::Base
     )
     expect(Test2Employee.where(comments: search_data)).to have_attributes(count: 1)
   end
+
+  it "should find NULL CLOB data when queried with nil" do
+    TestEmployee.delete_all
+    TestEmployee.create!(comments: nil)
+    TestEmployee.create!(comments: @char_data)
+    expect(TestEmployee.where(comments: nil)).to have_attributes(count: 1)
+  end
+
+  it "should find serialized NULL CLOB data when queried with nil" do
+    Test2Employee.delete_all
+    Test2Employee.create!(comments: nil)
+    Test2Employee.create!(comments: { some: 'text' })
+    expect(Test2Employee.where(comments: nil)).to have_attributes(count: 1)
```

Comment:
As far as I understand, this test should be using the `TestSerializedHashEmployee` or `TestSerializedHashEmployee2` class. Because it is named `... serialized ...`. Let me know if I didn't get your idea right.

Reply:
I think `TestEmployee2` or `TestSerializeEmployee` should work for this test case, as both have `serialize :comments`, but I switched it to `TestSerializeEmployee` to keep things matched up a little better.